.github/workflows/mobilecoin-workflow-dev-setup-environment.yaml

```diff
@@ -274,6 +280,229 @@ jobs:
         rancher_url: ${{ secrets.DEV_RANCHER_URL }}
         rancher_token: ${{ secrets.DEV_RANCHER_TOKEN }}
 
+    - name: Deploy PostgreSQL instance
+      uses: mobilecoinofficial/gha-k8s-toolbox@v1
+      with:
+        action: helm-deploy
+        chart_repo: https://charts.bitnami.com/bitnami
+        chart_name: postgresql
+        # chart_version: 13.1.5
+        chart_version: 11.9.13
+        chart_set: |
+          --set=global.postgresql.auth.existingSecret=fog-recovery-postgresql
+          --set=global.postgresql.auth.database=fog_recovery
+          --set=architecture=replication
+        chart_wait_timeout: 5m
+        release_name: fog-recovery-postgresql
+        namespace: ${{ inputs.namespace }}
+        rancher_cluster: ${{ secrets.DEV_RANCHER_CLUSTER }}
+        rancher_url: ${{ secrets.DEV_RANCHER_URL }}
+        rancher_token: ${{ secrets.DEV_RANCHER_TOKEN }}
+
+  setup-environment-v5-1:
+    # we want to match new builds "v0-" and "v5.1" or greater
+    if: ${{ ! startsWith(inputs.version, 'v5.0') }}
```

Comment:
Yes that's correct. 

We run 5.0.x charts to bootstrap config values to launch the bucket and db restore.  Since I'm pulling out the fog-services chart, I need a different path for launching "this run's chart" which start with the non-tag `v0` and run any `v5.1+` charts that might get deployed for later Alpha/Dev testing. 

I'm up for a better strategy, but the GHA logic and expressions are pretty primitive.